open-apparel-registry/open-apparel-registry
# Update footer for OS Hub

## Overview

This PR updates the site footer with the [design from N&S](https://codepen.io/niceandserious/pen/dyeOdmY/1a08d126f3479db9d98d55fda4539a74).

Connects #2079 
Closes #2123 
Closes #2174 

## Demo

![Screen Shot 2022-09-28 at 2 30 17 PM](https://user-images.githubusercontent.com/8356789/192860898-ed129770-187c-499b-ae3e-8306f483d57b.png)

## Notes

This PR changes the main content from `position: fixed` to `position: static`. This does affect the way the rest of the web page feels. It also fixes the bugs raised in issues #2123 and #2174.

The header needs an increased `z-index` now that it appears over the map while scrolling.


## Testing Instructions

* Go to the home page and verify the footer matches the design and the links work correctly
* Make sure the header looks correct on other pages too

## Checklist
